project_task_code_portal/readme/CONFIGURE.md

```diff
@@ -0,0 +1,10 @@
+# Configuration of Project Task Code Portal
+
+To configure the module, follow these steps:
+
+1. Go to **Apps** and install the **Project Task Code Portal** module.
```

Comment:
This instruction is located in the module description. Which means that user already has found this module in the "Apps".